go.sum

```diff
@@ -1,5 +1,6 @@
+4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
`go.sum` size diff is quite huge :worried:, looks linked to https://github.com/containers/image/pull/1597 and furthermore since [`github.com/sigstore/sigstore` dep](https://github.com/containers/image/pull/1597/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R26) has been added. Not sure I will merge this one.